### PR TITLE
🌱 Bump kpromo to v4.0.4

### PR DESCRIPTION
**Pull request by sbueringer**
**What this PR does / why we need it**:
Just saw that we're using an old version while debugging why kpromo failed on some k8s.io/k8s.io PR

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -742,10 +742,10 @@ $(SETUP_ENVTEST): # Build setup-envtest.
 	GOBIN=$(TOOLS_BIN_DIR) $(GO_INSTALL) $(SETUP_ENVTEST_PKG) $(SETUP_ENVTEST_BIN) $(SETUP_ENVTEST_VER)
 
 $(KPROMO):
```
Comment (sbueringer) [inline Makefile:744]: Apparently it worked before, just change dit for consistency

**Review by chrischdi — COMMENTED** [review]
/lgtm
/approve

**Comment by chrischdi** [comment]
@sbueringer to be cherry-picked, right?

**Comment by sbueringer** [comment]
> @sbueringer to be cherry-picked, right?

I'm not sure if we actually have to switch branches to generate the k8s.io PRs.

Independent of that. Can you maybe do a set of manual cherry-pick PRs to get the changes we made today and yesterday in our Makefiles on all branches (e.g. the generate-release-notes change)

**Pull request merged**